CMakeLists.txt

```diff
@@ -163,43 +162,53 @@ if(AUTO_PLUGIN_DEPLOYMENT OR AIO_ZIP_TO_DIST)
 		COMMAND ${CMAKE_COMMAND} -E copy $<TARGET_PDB_FILE:${PROJECT_NAME}> "${AIO_DIR}/SKSE/Plugins/"
 	)
 
+	# Generate timestamps for shader files
+	file(GLOB_RECURSE HLSL_FILES "${CMAKE_SOURCE_DIR}/package/*.hlsl*" "${FEATURE_PATHS}/*.hlsl*")
+	file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/shader_timestamps)
+
+	foreach(file IN LISTS HLSL_FILES)
+		get_filename_component(FILE_NAME ${file} NAME)
+		set(TIMESTAMP_FILE "${CMAKE_BINARY_DIR}/shader_timestamps/${FILE_NAME}.timestamp")
```

Comment:
Shouldn't this also create a folder to match the directory tree in Shaders? Otherwise this could cause duplicate timestamps if we for example have multiple `Common.hlsli` in different folders? So maybe should not generate a flat list?
<img width="304" alt="image" src="https://github.com/user-attachments/assets/c3a178b9-0252-443f-aff5-546edc5c5ebb">